httpfuzzerprocessor/RTT_time_filter.py

```diff
@@ -0,0 +1,44 @@
+# Version 1.0
+# @author RUFFENACH Timothée
+# filter by RTT (time request).
+
+from javax.swing import JFrame, JPanel, JComboBox, JOptionPane,JFileChooser,JOptionPane
+
+# Auxiliary variables/constants needed for processing.
+global time,isCheck;
+init = False
+
+def getNumber(min,max,asked):
+	number = JOptionPane.showInputDialog(None, asked, "Input", JOptionPane.QUESTION_MESSAGE)
+    
+	if int(number) >= min and int(number) <= max:
+		number = int(number)
+		return number
+	else:
+		JOptionPane.showMessageDialog(None, "Choose number between " +  min  + " to " + max)
+		getNumber()
+
+# Called after injecting the payloads and before forward the message to the server.
+def processMessage(utils, message) :
+	if (init == False):
+		initialise()
+
+def initialise():
+	global init,entry,isCheck
+	global time;
+	time = getNumber(1,50000,"how many time do you want ?")
+	isCheck = JOptionPane.showConfirmDialog(None, "more high or equal (YES) esle less or equal (NO)", "Confirm", JOptionPane.YES_NO_OPTION)
```

Comment:
This English is rough, I'm happy to help but I need a better description of what's meant here.

They should both start with capitals and there's a type in "else".

Reply:
I changed the sentences for more understanding, sorry my native language is not english

Reply:
No problem, that's why I was offering to help 👍